Jenkinsfile

```diff
@@ -8,6 +8,7 @@
 def images = [
     [image: "docker.io/library/amazonlinux:2",          arches: ["aarch64"]],
     [image: "quay.io/centos/centos:stream9",            arches: ["amd64", "aarch64"]],          // CentOS Stream 9 (EOL: 2027)
+    [image: "quay.io/centos/centos:stream10-development", arches: ["amd64", "aarch64"]],        // CentOS Stream 10 (EOL: 2030)
```

Comment:
keeping the columns mis-aligned for now, as the tag will be renamed after release